src/screens/catalog-page/catalog-page.js

```diff
@@ -1,86 +1,53 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import { Split, Article } from 'grommet';
-import { sidebarHide } from '../../actions/sidebar';
-import FilterSidebar from '../../components/filter-sidebar/filter-sidebar';
+import Sidebar from '../../components/sidebar/sidebar';
 import TilesComponent from '../../components/tiles/tiles';
 import CatalogPageTile from '../../components/tiles/tile/catalog-page_tile';
 
-const items = [
-  {
-    key: '1',
-    img: 'https://goo.gl/RWPJEy',
-    brand: 'nike',
-    name: 'Air Mag',
-    price: 45000,
-  },
-  {
-    key: '2',
-    img: 'https://goo.gl/RWPJEy',
-    brand: 'nike',
-    name: 'Air Mag',
-    price: 45000,
-  },
-  {
-    key: '3',
-    img: 'https://goo.gl/RWPJEy',
-    brand: 'nike',
-    name: 'Air Mag',
-    price: 45000,
-  },
-  {
-    key: '4',
-    img: 'https://goo.gl/RWPJEy',
-    brand: 'nike',
-    name: 'Air Mag',
-    price: 45000,
-  },
-  {
-    key: '5',
-    img: 'https://goo.gl/RWPJEy',
-    brand: 'nike',
-    name: 'Air Mag',
-    price: 45000,
-  },
-  {
-    key: '6',
-    img: 'https://goo.gl/RWPJEy',
-    brand: 'nike',
-    name: 'Air Mag',
-    price: 45000,
-  },
-  {
-    key: '7',
-    img: 'https://goo.gl/RWPJEy',
-    brand: 'nike',
-    name: 'Air Mag',
-    price: 45000,
-  },
-  {
-    key: '8',
-    img: 'https://goo.gl/RWPJEy',
-    brand: 'nike',
-    name: 'Air Mag',
-    price: 45000,
-  },
-];
+import items from '../../assets/items';
 
 const CatalogPage = props => {
-  const { sidebar: { enabled } } = props;
+  const {
+    sidebar: { enabled },
+    filter: { values },
+    match: { params },
+    location: { state },
+  } = props;
+  let tiles = items;
```

Comment:
First, it's probably not a good performance-wise choice to do .filter so many times. You should have one function that can check everything that is required and then do .filter once.
Second, that logic should probably be located inside a reducer, not in the component render